src/codec_avm.c

```diff
@@ -460,7 +460,7 @@ static avifResult avmCodecEncodeImage(avifCodec * codec,
 {
     struct aom_codec_enc_cfg * cfg = &codec->internal->cfg;
     avifBool quantizerUpdated = AVIF_FALSE;
-    int quantizer = avmQualityToQuantizer(quality, image->depth);
+    const int quantizer = avmQualityToQuantizer(quality, image->depth);
```

Comment:
IMPORTANT: Remove this `const`. Instead, add it to the `quantizer` local variable inside `avmQualityToQuantizer()` (line 443).

This variable will be modified at 589 below:

```
        // Scale from aom's [0:63] to avm's [0:255]. TODO(yguyon): Accept [0:255] directly in avifEncoder.
        quantizer = avmScaleQuantizer(quantizer, image->depth);
```